.github/workflows/test-e2e.yml

```diff
@@ -20,9 +20,9 @@ jobs:
         name: Compute Test Matrix
         run: |
           # Note the use of external single quotes to allow for double quotes at inline YAML array
-          matrix='["v1.29.8-kind"]'
+          matrix='["v1.30.8-kind"]'
           if [ "${{ github.ref }}" == "refs/heads/main" ];then
-            matrix='["v1.29.8-kind", "v1.31.1-kind"]'
+            matrix='["v1.30.8-kind", "v1.32.0-kind"]'
```

Comment:
According to our support policy, this is not correct. 1.32 is out since 2024 Dec 11, the 2 month buffer is not reached yet.